src/Microsoft.Azure.SignalR.Management/ServiceManagerOptions.cs

```diff
@@ -25,30 +23,72 @@ public class ServiceManagerOptions
 
         /// <summary>
         /// Gets or sets the connection string of Azure SignalR Service instance.
+        /// Gets or sets the connection string of Azure SignalR Service instance and switches to single-endpoint mode.
         /// </summary>
-        public string ConnectionString { get; set; } = null;
+        public string ConnectionString
+        {
+            get => _connectionString;
+            set
+            {
+                if (string.IsNullOrWhiteSpace(value))
```

Comment:
Is it possible to put all the checks into `internal void ValidateOptions()`?

Reply:
I think it's better to validate new value of each property in its own setter, otherwise the `ValidateOptions()` would be big and has to deal with many conditions. What about only moving transport mode conflict detect to `ValidateOptions()`?